webpack.config.dev.js

```diff
@@ -1,15 +1,13 @@
-const path = require('path');
-
-const PROJECT_PATH = path.join(__dirname, '/local_test');
```

Comment:
I think this is related to local manual testing? If so I don't think it can be deleted.